third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -2023,6 +2023,26 @@ func TestComputeInstance_networkIPCustomizedDiff(t *testing.T) {
 	}
 }
 
+<% unless version == 'ga' -%>
+func TestAccComputeInstance_multiWriter(t *testing.T) {
```

Comment:
This test should be in Disk because it's primarily a disk test, right?

Reply:
yep yep :)